forte/data/data_pack.py

```diff
@@ -244,19 +248,22 @@ def _validate(self, entry: EntryType) -> bool:
         return isinstance(entry, SinglePackEntries)
 
     @property
-    def text(self) -> str:
-        r"""Return the text of the data pack"""
-        return self._text
+    def text(self, text_payload_index: int = 0) -> str:
```

Comment:
We can remove the argument `text_payload_index` here since there is no way to pass argument to a getter method decorated with `@property`.

Reply:
OK. Should I return the text from all text payloads or just the first text payload? I am thinking maybe all text payloads?

Reply:
For backward compatibility, I think it's better to just return the text from the first text payload.